source/tutorial/model-hook.md

```diff
@@ -0,0 +1,68 @@
+Now, let's add a list of available rentals to the index template. We know that rentals will not be static: eventually, users will be able to add, update or delete them.  For this reason, we will need a _rentals_ model to save information about the rentals. However, to keep things simple at first, our model is just going to be a hard-coded array of JavaScript objects. Later, we'll switch to using Ember Data, so that we can let users create and modify their own rentals.
```

Comment:
> that rentals will not be static: eventually, users will be able ...

The use of a colon there bothers me (can't quite put my finger on why).  Can we replace it with a `for`?